src/declarativewebutils.cpp

```diff
@@ -43,6 +47,19 @@ bool testScreenDimensions(qreal pixelRatio) {
     return fmod(w, 1.0) == 0 && fmod(h, 1.0) == 0;
 }
 
+const QSettings &quickSettings()
+{
+    static const QSettings settings(QSettings::SystemScope, QStringLiteral("QtProject"), QStringLiteral("QtQuick2"));
```

Comment:
No need for 'static' here; you only access the settings once, so it won't be helpful to keep them in memory.